native/packages/react-native-bpk-component-badge/src/styles.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StyleSheet } from 'react-native';
+
+import {
+  colorGray700,
+  colorGray50,
+  colorYellow500,
+  colorRed500,
+  colorGreen500,
+  colorWhite,
+  borderRadiusSm,
+  borderSizeSm,
+  spacingSm,
+  spacingMd,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import { setOpacity } from 'bpk-tokens';
+
+export const styles = {
+  view: StyleSheet.create({
+    base: {
+      paddingHorizontal: spacingMd,
+      paddingVertical: spacingSm,
+    },
+    warning: {
+      backgroundColor: colorYellow500,
+    },
+    success: {
+      backgroundColor: colorGreen500,
+    },
+    destructive: {
+      backgroundColor: colorRed500,
+    },
+    light: {
+      backgroundColor: colorGray50,
+    },
+    inverse: {
+      backgroundColor: colorWhite,
+    },
+    outline: {
+      paddingHorizontal: spacingMd - borderSizeSm,
```

Comment:
when the badge is outlined the border is substracted from the padding in order to have outlined and not outlined badges aligned with each other